modules/actions/disconnect.js

```diff
@@ -18,6 +18,10 @@ import { osmNode } from '../osm/node';
 export function actionDisconnect(nodeId, newNodeId) {
     var wayIds;
 
+    var disconnectableRelationTypes = {
+        'site': true,
+        'assosciatedStreet': true
+    };
```

Comment:
Other potentially disconnectable relation types:

* [`enforcement`](https://wiki.openstreetmap.org/wiki/Relation:enforcement) – specifically, the `device` member
* [`building`](https://wiki.openstreetmap.org/wiki/Relation:building) – not about routing, but it should probably be possible to disconnect a building outline from something unrelated like an adjoining footway

Reply:
> it should probably be possible to disconnect a building outline from something unrelated like an adjoining footway

yes, but this is not prevented by the disconnect action: only members of the _same_ relation are kept together, so that the existing connectivity within those relations are always conserved. Objects which are part of different relations (or situations where only one object is member of any relations) are (and were always) allowed to be disconnected.

> `enforcement`

:+1: this sound like a good addition in order to be able to unglue accidentally snapped enforcement devices from their streets